src/test/java/com/logigear/RegisterTest.java

```diff
@@ -26,4 +26,27 @@ public void TC07() {
         String expectedMsg = "You're here";
         Assert.assertEquals(actualMsg, expectedMsg, "Error message fails to display!");
     }
+
+    @Test(description = "TC11 - User can't create account while password and PID fields are empty")
+    public void TC11() {
+        LoggerHelper.startTestCase("TC11 - User can't create account while password and PID fields are empty");
+
+        String blankPassword = "";
```

Comment:
again? I already commented about this in this PR: https://github.com/sattdn21-01-02/selenium-minh2/pull/9
and now you do that again? why?

Reply:
sorry that my fault, I will never repeat  this error again